src/Jade/Compiler.php

```diff
@@ -708,7 +714,7 @@ protected function createStatements()
         if (count($statements) == 0) {
             $php_string = call_user_func_array('sprintf', $code_format);
 
-            return '<?php ' . $php_string . ' ?>';
+            return '<?php ' . $php_string . " {$this->closingTag}";
```

Comment:
Hum, I dislike to mix concat and interpolation in the same line. Would prefer:
```php
return '<?php ' . $php_string . ' ' . $this->closingTag;
```
And same for other lines.